go/registry/api/api.go

```diff
@@ -281,23 +282,64 @@ type EntityEvent struct {
 	IsRegistration bool           `json:"is_registration"`
 }
 
+// EventKind returns a string representation of this event's kind.
+func (e *EntityEvent) EventKind() string {
+	return "entity_event"
```

Comment:
Why the `_event` suffix? Should probably remove it to be consitent with other places.